api/scanner/media_encoding/media_utils/photo_dimensions.go

```diff
@@ -48,6 +48,11 @@ func (dimensions *PhotoDimensions) ThumbnailScale() PhotoDimensions {
 		height = 1024
 	}
 
+	if width > dimensions.Width {
+		width = dimensions.Width
+		height = dimensions.Height
+	}
```

Comment:
I'd appreciate if you send another PR for this change.
Should it also consider if the height is smaller?

Reply:
Because the aspect of thumbnail is the same as photo, comparing width or height is enough. I choose comparing width.

Reply:
I move this commit to a new pull request #798, and this one will be closed.